docker/Dockerfile

```diff
@@ -0,0 +1,50 @@
+FROM debian:11.6
+
+ARG JOBS=1
+
+ENV TZ America/Sao_Paulo
```

Comment:
I brought it from the Date and Time IOC. I though of keeping it because of timestamped logs and events.

Maybe this should go to our standardization discussion?

Reply:
Sounds reasonable!